patches/electron-updater+5.3.0.patch

```diff
@@ -1,25 +0,0 @@
-diff --git a/node_modules/electron-updater/out/windowsExecutableCodeSignatureVerifier.js b/node_modules/electron-updater/out/windowsExecutableCodeSignatureVerifier.js
```

Comment:
It looks like we'll still need to patch this issue because the latest version of `electron-updater` still has this problem https://github.com/electron-userland/electron-builder/blob/1af7447edf47303de03ca2924727c78118161c60/packages/electron-updater/src/windowsExecutableCodeSignatureVerifier.ts#L42-L46

Reply:
> It looks like we'll still need to patch this issue because the latest version of `electron-updater` [still has this problem](https://github.com/electron-userland/electron-builder/blob/1af7447edf47303de03ca2924727c78118161c60/packages/electron-updater/src/windowsExecutableCodeSignatureVerifier.ts#L42-L46)

Do you mind lending me a hand on this one? I am not very familiar with git patches

Reply:
Just added it back